cypress/steps/productSteps.js

```diff
@@ -0,0 +1,46 @@
+import { PRODUCTS_SELECTORS } from "../elements/catalog/product-selectors";
+
+class ProductSteps {
+  updateProductIsAvailableForPurchase(productUrl, isAvailableForPurchase) {
+    let isAvailableForPurchaseSelector;
```

Comment:
`lets` complicate code flow, so I recommend avoiding them alltogether. Also, since these selectors get pretty long, why not destructure it:
```suggestion
    const { radioButtonsValueTrue, radioButtonsValueFalse, -all others used in this file } = PRODUCT_SELECTORS;
    
    const isAvailableForPurchaseSelector = isAvailableForPurchase ? radioButtonsValueTrue : 
      radioButtonsValueFalse
```